app/src/main/res/layout/activity_descripstion.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".DescripstionActivity">
+
+    <ImageView
+        android:id="@+id/image_hero"
+        android:layout_width="match_parent"
```

Comment:
not match_parent. vừa trình bày xong constrainlayout mà. tương tự vs các view khác.